components/brave_new_tab_ui/stories/default/data/todayStorybookState.ts

```diff
@@ -0,0 +1,957 @@
+import { boolean } from '@storybook/addon-knobs/react'
+import { BraveTodayState } from '../../../reducers/today'
+
+export default function getTodayState (): BraveTodayState {
+  return {
+    isFetching: boolean('Today is fetching?', false),
+    isUpdateAvailable: boolean('Is Today update available?', false),
+    currentPageIndex: 10,
+    publishers: {
+      ['5eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5']: {
+        publisher_id: `5eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5`,
+        publisher_name: `Test Publisher 1`,
+        category: `Tech`,
+        enabled: false,
+        user_enabled: true
+      },
+      ['4eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5']: {
+        publisher_id: `4eece347713f329f156cd0204cf9b12629f1dc8f4ea3c1b67984cfbfd66cdca5`,
+        publisher_name: `Test Publisher 2`,
+        category: `Top News`,
+        enabled: false,
+        user_enabled: null
+      }
+    },
+    feed: {
+      hash: `123abc`,
+      featuredArticle: {
+        category: `Top News`,
+        publish_time: `2020-10-22 00:09:00 UTC`,
+        url: `https://www.npr.org/2020/10/21/926445682/u-s-blames-iran-for-threatening-election-emails-says-russia-may-interfere-too?utm_medium=RSS&utm_campaign=news`,
+        img: `https://pcdn.brave.software/brave-today/cache/9bbfe86e499867a3988d7b089daa757cb7762a6a9d9f3b9836c4f5b8d8cd610b.jpg.pad`,
```

Comment:
The `img` field is no longer included in the feed (https://github.com/brave-intl/brave-today-aggregator/pull/20).